### ISSUE TITLE
[Product Pull Request] fix: Fix rows counter in the Edit Grade modal window

### ISSUE THREAD

**Issue by jmakowski1123** [openedx/platform-roadmap#236]
### For Contributing Author:

This is the Primary Product Ticket for the following community contribution: Make course description editable in certificates

### Checklist prior to undergoing Product Review:

The following information is required in order for Product Managers to be able to review your pull request:

- [x] Explanation of the problem being solved
- [ ] Description of how users will be impacted, and which users will be impacted
- [x] Screenshots or video showing the functionality or fix, before and after
- [ ] Reproduction steps and/or testing steps

Only if necessary:

- [ ] If necessary, links to corresponding configuration changes
- [ ] If necessary, links to corresponding enablement changes, particularly waffle/toggle status details

### Related PRs

- [ ] https://github.com/openedx/frontend-app-gradebook/pull/310
- [ ] https://github.com/openedx/frontend-app-gradebook/pull/311

### For Product Manager doing the review:

What criteria should be analyzed from Product to approve a PR?

- The problem being solved by the feature or fix is clear.
- There is clarity on how the change or fix will impact the end user.
- It is clear that the change will not negatively impact users or other areas of the platform.
- The change is implemented comprehensively.
- Any changes to UI use the current, standard Paragon Design System: https://paragon-openedx.netlify.app/

**Comment by jmakowski1123** [comment]
Original information from the PR:

**TL;DR -** The problem was in the rows counter in the Edit Grades modal window. First digit - number of lines excluding the last line with the form. Second digit - grades data. And our proposal is to include last row with form to common counting

<img width="1904" alt="Снимок экрана 2023-02-10 в 15 25 06" src="https://user-images.githubusercontent.com/19806032/218103195-661c7642-72cb-49bd-be9a-bfe7f8fd0eb3.png">

**What changed?**

<img width="1904" alt="Снимок экрана 2023-02-10 в 15 42 01" src="https://user-images.githubusercontent.com/19806032/218106655-0a52059c-4041-4c5d-935a-9c89b8183bc2.png">

FYI: @openedx/content-aurora

**Comment by jmakowski1123** [comment]
@spencertiberi This is a set of PRs to fix a bug in the grading modal. It seems pretty straightforward to me, but let me know if you need or want further information before giving it a product review. Once you've completed your review, leave a comment here that the review is done!

**Comment by mphilbrick211** [comment]
@spencertiberi @ProductRyan just checking in to see if there's an update on the product review side?

**Comment by Daniel-hershel** [comment]
Hi everyone, I'm Daniel the new Product Manager with the Aurora team. I'll be taking over the product review responsibility for Aurora from Ryan. This PR looks good to me, thanks for the contribution!

**Comment by mphilbrick211** [comment]
Thanks @Daniel-hershel - just confirming that product review is complete on both: https://github.com/openedx/frontend-app-gradebook/pull/310 and https://github.com/openedx/frontend-app-gradebook/pull/311

**Comment by Daniel-hershel** [comment]
Hi @mphilbrick211 , sorry I didn't realize there were two here, but they look the same to me from a product/user experience perspective. Am I missing anything that delineates the two? Thanks!

**Comment by mphilbrick211** [comment]
Hi @Daniel-hershel - looks like one is Olive branch-specific.

**Comment by Daniel-hershel** [comment]
Can confirm this has completed product review for both tickets associated with this issue

**Comment by sarina** [comment]
Closing as this is marked "Shipped" on the PR board 🎉
